pages/api/v1/auth/actions/checkFollowStatus.ts

```
@@ -0,0 +1,41 @@
+import getTokenFromReq from '~src/auth/utils/getTokenFromReq';
+
+interface FollowStatusResponseType {
+	message: string;
+	isFollowing: boolean;
+}
+
+async function handler(req: NextApiRequest, res: NextApiResponse<FollowStatusResponseType>) {
+	storeApiKeyUsage(req);
+
+	const { userIdToCheck } = req.query;
+	if (!userIdToCheck || isNaN(Number(userIdToCheck))) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)